.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] As the previous reviewer requested, please add `/node_modules` directory in  `.gitignore` file. It is necessary to list the `node modules` directory in `.gitignore` because it can be added by accident while setting up Linters.